pyscript.core/esm/interpreter/_utils.js

```diff
@@ -131,3 +131,12 @@ export const fetchPaths = (module, interpreter, config_fetch) =>
                 .then((buffer) => module.writeFile(interpreter, path, buffer)),
         ),
     );
+
+// this is a fallback for interpreters unable to register JS modules
+// all defined keys will end up as globally available references
+// REQUIRES INTEGRATION TEST
+/* c8 ignore start */
+export function registerJSModule(interpreter, _, value) {
+    for (const [k, v] of entries(value)) this.setGlobal(interpreter, k, v);
+}
+/* c8 ignore stop */
```

Comment:
@fpliger see here

Reply:
Yeah, exactly. Saw that after I left my comments... but I think it's worth spelling out the intentions anyway just to make sure we are explicitly in agreement (without having to check the implementation :) )

Reply:
the intention is written in the code and, as mentioned in the previous comment, there's no other way to do the same in foreign, not fully capable, interpreters (unfortunately ... or due my not 100% understanding of their internals).